api/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
add some tests or at least list what tests might be useful to have as a todo comment.